modules/StdEnv/2023.lua

```diff
@@ -0,0 +1,35 @@
+add_property(   "lmod", "sticky")
+
+require("os")
+load("CCconfig")
+load("gentoo/2023")
+
+if (mode() == "spider") then
+	-- set by gentoo/2023 module
+	local arch = os.getenv("RSNT_ARCH")
```

Comment:
I don't think that's defined at this point if it's set in gentoo/2023, the environment is not modified until after all modules are loaded, I think.

Reply:
this logic is simply copy/pasted from StdEnv/2020.lua, it's just to let `module spider` show StdEnv/2023 instead of gentoo/2023, so it should work the same way as in 2020.

Reply:
ok then, I guess it works just as well (or as bad(?)) as StdEnv/2020. Any way, good enough :)